modules/ocf_hpc/manifests/controller.pp

```diff
@@ -24,4 +24,15 @@
       File['/etc/slurm-llnl/cgroup.conf']
     ],
   }
+
+  # Set up script to autoadd LDAP group members to SLURM.
+  file { '/usr/local/bin/add_slurm_users':
+    source => 'puppet:///modules/ocf_hpc/add_slurm_users.py',
```

Comment:
Removing the extension on this file (within the puppet repo) is probably best, since it's not in the added file, so it keeps things more consistent.